python/src/vmaf/core/quality_runner.py

```diff
@@ -420,6 +420,137 @@ def _remove_result(self, asset):
         vmaf_fassembler = self._get_vmaf_feature_assembler_instance(asset)
         vmaf_fassembler.remove_results()
 
+class EnsembleVmafQualityRunner(VmafQualityRunner):
```

Comment:
We can put this in for now, but may need to revisit later to refactor it to make the structure better.